src/app/static/src/app/components/plots/Choropleth.vue

```diff
@@ -90,14 +86,9 @@
             ),
             ...mapStateProps<FilteredDataState, keyof FilteredDataComputed>("filteredData", {
                     selectedDataType: state => state.selectedDataType,
-                    selectedRegions: state => state.selectedChoroplethFilters.regions || []
+                    selectedFilters: state => state.selectedChoroplethFilters
                 }
             ),
-            ...mapGettersByNames<keyof FilteredDataGetters>("filteredData", [
-                    "regionIndicators",
-                    "colorFunctions"
```

Comment:
The computed `regionIndicators`  has gone, we now manually update region colours when anything changes. This `colorFunctions` wasn't being used anywhere.